rust/src/dcerpc/dcerpc.rs

```diff
@@ -1378,10 +1378,9 @@ pub const PARSER_NAME: &'static [u8] = b"dcerpc\0";
 
 #[no_mangle]
 pub unsafe extern "C" fn rs_dcerpc_register_parser() {
-    let default_port = CString::new("[0:65355]").unwrap();
     let parser = RustParser {
         name: PARSER_NAME.as_ptr() as *const std::os::raw::c_char,
-        default_port: default_port.as_ptr(),
+        default_port: b"\0".as_ptr() as *const std::os::raw::c_char,
```

Comment:
This is creating an empty string, not null.  `std::ptr::null()` can be used here for a real null value.

Reply:
Thanks. I thought this was null...